### PR TITLE
change: [OCA-1240] - Update Main Concept app names to include demo

### PR DESCRIPTION
**Pull request by tbaka**
## Description 📝
Updates app names to highlight deployments are demos.

## Changes  🔄
updates stasckscript.utils
updates oneClickApps.ts

### CODE REVIEW

**Comment by tyler-akamai** [comment]
make sure to add changeset: 
`yarn changeset`

**Review by hana-linode — APPROVED** [review]

**Review by cpathipa — APPROVED** [review]

**Review by tyler-akamai — APPROVED** [review]

File: packages/manager/src/features/OneClickApps/oneClickApps.ts [hunk 1]
```diff
@@ -1173,15 +1173,15 @@ export const oneClickApps: OCA[] = [
...
-    name: 'MainConcept P2 AVC ULTRA Transcoder',
+    name: 'MainConcept P2 AVC Ultra Transcoder Demo',
     related_guides: [
       {
         href:
-          'https://www.linode.com/docs/products/tools/marketplace/guides/mainconcept-p2-avc-ultra/',
+          'https://www.linode.com/docs/products/tools/marketplace/guides/mainconcept-p2-avc-ultra-demo/',
         title:
-          'Deploy MainConcept P2 AVC-ULTRA Transcoder through the Linode Marketplace',
+          'Deploy MainConcept P2 AVC Ultra Transcoder Demo through the Linode Marketplace',
```
Comment (mjac0bs) [inline packages/manager/src/features/OneClickApps/oneClickApps.ts:1184]: @tbaka A question on naming: these changes replace the update made in an earlier PR that preferred "ULTRA" over "Ultra" based on this [comment](https://github.com/linode/manager/pull/9858#issuecomment-1803227205). We've had a  few name changes with these apps, so I wanted to check if this revert was intentional.

Comment (tbaka) [reply packages/manager/src/features/OneClickApps/oneClickApps.ts]: Thanks for pointing that out, I was not aware of that, and the change in casing was unintentional.

**Review by mjac0bs — APPROVED** [review]
Looks good to me now! Confirmed that all the MainConcept app names display 'Demo' and all the drawers open and display correctly on this branch.

🚀

**Pull request merged**